Evgeniyyyyy/PlatformaticaQA_Yevhen
# Yb test Chain1 Test Create new record() added

1. Chain1 Test Create new record() added
AT730.01 Java Entity>Events Chain1: Create new record

Test Case added:
PreCondition:
User login to http:    with vaid username and login credentials

Steps:
Go to http: https://ref2.eteam.work/index.php?action=action_list&entity_id=61&mod=2
2. Click on the red button located at left top of the screen
3. Input int F1 = 1
4. Wait until values F2-F10 avtomaticaly created
5. Click on the Save button

Expected result: The appropriate record will be created and displayed in the Event Chain page. Values for F1=1 are 1, 2, 4, 8, 16, 32, 64, 128, 256, 512.
Check box is present
